lineman/app/components/edit_group_form/edit_group_form.coffee

```diff
@@ -6,9 +6,14 @@ angular.module('loomioApp').factory 'EditGroupForm', ->
     $scope.submit = ->
       $scope.group.save().then ->
         $scope.$close()
-        FlashService.success 'group_form.edit_group_success'
-      , ->
-        $rootScope.$broadcast 'pageError', 'cantEditGroup', $scope.group
+        FlashService.success 'edit_group_form.messages.success'
```

Comment:
@gdpelican, yeah it seems pretty necessary. Think it's a separate ticket though - maybe you wanna stick one in the sprint board with your suggestion?